CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2021-07-03
+* Added new VFX 2022 Draft images with the following versions:
+    * python-3.9.5
+    * tbb-2020_U3
+    * boost-1.75.0
+    * cppunit-1.15.1
+    * log4cplus-1.1.2
+    * glew-2.1.0
+    * glfw-3.1.2
+    * qt-5.15.2
+    * pyside-5.15.2
+    * cmake-3.20.5
+    * openexr-2.5.5
+    * blosc-1.5.0
+    * alembic-1.8.2
+    * oiio-2.2.14.0
```

Comment:
oiio-2.2.15.0 just released, although it doesn't matter a ton; it's more important that we update ~Sept 1 when 2.3 is released.

Reply:
Thanks, I'll be working on the better oiio/ocio packages next so I'll definitely pick it up soon.